CodeMaid/CodeMaid.csproj

```diff
@@ -340,12 +340,13 @@
       <Version>1.0.2</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.VisualStudio.SDK">
-      <Version>16.10.31321.278</Version>
+      <Version>17.6.36389</Version>
```

Comment:
I believe these major package updates will make the extension incompatible with VS2019.